code/modules/antagonists/revolution/revolution.dm

```diff
@@ -253,17 +252,9 @@
 		H.dna.remove_mutation(CLOWNMUT)
 
 	if(give_flash)
-		var/obj/item/assembly/flash/handheld/T = new(H)
-		var/list/slots = list (
-			"backpack" = SLOT_IN_BACKPACK,
-			"left pocket" = SLOT_L_STORE,
-			"right pocket" = SLOT_R_STORE
-		)
-		var/where = H.equip_in_one_of_slots(T, slots)
-		if (!where)
-			to_chat(H, "The Syndicate were unfortunately unable to get you a flash.")
-		else
-			to_chat(H, "The flash in your [where] will help you to persuade the crew to join your cause.")
+		var/obj/item/organ/cyberimp/arm/flash/rev/T = new
+		T.Insert(H, special = TRUE, drop_if_replaced = FALSE)
+		to_chat(H, "The flash implant in your arm will help you to persuade the crew to join your cause.")
```

Comment:
some input saying they can only use this to convert and bigger text font would help